plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAlluxioCacheFileOperations.java

```diff
@@ -154,17 +154,17 @@ public void testCacheFileOperations()
         assertFileSystemAccesses(
                 "SELECT * FROM test_cache_file_operations",
                 ImmutableMultiset.<CacheOperation>builder()
-                        .add(new CacheOperation("Alluxio.readCached", "00000000000000000000.json", 0, 794))
+                        .add(new CacheOperation("Alluxio.readCached", "00000000000000000000.json", 0, 816))
                         .add(new CacheOperation("InputFile.length", "00000000000000000000.json"))
-                        .add(new CacheOperation("Alluxio.readCached", "00000000000000000001.json", 0, 636))
+                        .add(new CacheOperation("Alluxio.readCached", "00000000000000000001.json", 0, 658))
                         .add(new CacheOperation("InputFile.length", "00000000000000000001.json"))
-                        .add(new CacheOperation("Alluxio.readCached", "00000000000000000002.json", 0, 636))
+                        .add(new CacheOperation("Alluxio.readCached", "00000000000000000002.json", 0, 658))
                         .add(new CacheOperation("InputFile.length", "00000000000000000002.json"))
-                        .add(new CacheOperation("Alluxio.readCached", "00000000000000000003.json", 0, 636))
+                        .add(new CacheOperation("Alluxio.readCached", "00000000000000000003.json", 0, 658))
                         .add(new CacheOperation("InputFile.length", "00000000000000000003.json"))
-                        .add(new CacheOperation("Alluxio.readCached", "00000000000000000004.json", 0, 636))
+                        .add(new CacheOperation("Alluxio.readCached", "00000000000000000004.json", 0, 658))
                         .add(new CacheOperation("InputFile.length", "00000000000000000004.json"))
-                        .add(new CacheOperation("Alluxio.readCached", "00000000000000000005.json", 0, 636))
+                        .add(new CacheOperation("Alluxio.readCached", "00000000000000000005.json", 0, 658))
```

Comment:
Could you fix `TestDeltaLakeAlluxioCacheFileOperations.testCreateOrReplaceTable` and `TestDeltaLakeAlluxioCacheFileOperations.testCreateOrReplaceTableAsSelect`?